specification/synapse/resource-manager/Microsoft.Synapse/stable/2021-06-01/bigDataPool.json

```
@@ -575,6 +580,7 @@
           "description": "The minimum number of executors alloted",
           "x-ms-mutability": [
             "create",
+            "read",
             "update"
           ]
```

Rather than adding "read" you should just remove x-ms-mutability altogether.
```suggestion
          "description": "The minimum number of executors alloted"
```